examples/fluids/problems/newtonian.c

```diff
@@ -265,6 +265,8 @@ PetscErrorCode NS_NEWTONIAN_IG(ProblemData *problem, DM dm, void *ctx) {
   newtonian_ig_ctx->state_var     = state_var;
   PetscCall(PetscArraycpy(newtonian_ig_ctx->g, g, 3));
 
+  PetscCall(FreestreamBCSetup(problem, dm, ctx, newtonian_ig_ctx));
```

Comment:
Tests are failing because `-freestream_velocity` isn't given for the other examples. Should this only be called when there's at least one `-bc_freestream` (`bc->num_freestream > 0`)?

Reply:
Yeah, saw that. I was just going to add in the `set` boolean and check there. Running for `bc->num_freestream > 0` would be ideal, but that requires piping `bc` into every problem configuration function (otherwise it's not available).

Reply:
I'm open to whichever. You have an opinion either way?

Reply:
Where can we ensure that if there exists a `-bc_freestream`, that these values have been set? (I don't really care about velocity because it can be zero, but pressure and temperature need to be set. I guess we could have STP defaults.

Reply:
I think `newtonian.c` makes the most sense for that location. It's called right after the command line arguments are processed, and the freestream BCs are only defined for Newtonian right now.

Reply:
Sounds good.